dbms/src/Common/RadixSort.h

```diff
@@ -179,8 +198,101 @@ struct RadixSort
     static KeyBits keyToBits(Key x) { return ext::bit_cast<KeyBits>(x); }
     static Key bitsToKey(KeyBits x) { return ext::bit_cast<Key>(x); }
 
+    static inline void insertSortInternal(Element * arr, size_t size)
+    {
+        for (Element * i = arr + 1; i < arr + size; ++i)
```

Comment:
Need to check if `arr + size` is calculated in every loop iteration (it is possible due to aliasing).

Reply:
We must check if `std::sort` or `pdqsort` will be better.

Reply:
Created a pre-computed variable to ensure this never happens.